package.json

```diff
@@ -29,9 +29,9 @@
     "bs-platform": "6 || 7 || 8"
   },
   "devDependencies": {
-    "@glennsl/bs-jest": "^0.5.1",
-    "bs-platform": "8",
+    "@glennsl/bs-jest": "0.6.0",
+    "bs-platform": "8.4.2",
```

Comment:
Updated this 2, since I got a nasty error with typemodel.ml and updating made the error clearer. Happy to take this out of this PR.

Reply:
Just to make sure I understand – the reason for upgrading `bs-platform` was so that while developing it would give you better error messages? I want to make sure we're not breaking compatibility with `bs-platform` < `8.4.2`

Reply:
Right, I can revert it since was just one-of. But the error was coming from bs-jest and `typemodel.ml` and was very cryptic, updating gave the exact Location and after updating bs-jest everything worked fine (Don't have the error in hand, but If I revert to bs-platform 8, I can try to repro)